packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -38,9 +38,7 @@ function BlockListAppender( {
 	if ( CustomAppender ) {
 		// Prefer custom render prop if provided.
 		appender = <CustomAppender />;
-	} else if ( canInsertDefaultBlock ) {
-		// Render the default block appender when renderAppender has not been
-		// provided and the context supports use of the default appender.
+	} else {
```

Comment:
All the changes in this file are just copied from #25518 for the sake of testing. Please review that PR and help it get merged.